cost-analyzer/values.yaml

```diff
@@ -268,6 +268,8 @@ kubecostModel:
   # The total number of hours the ETL pipelines will build
   # Set to 0 to disable hourly ETL (not recommended)
   etlHourlyStoreDurationHours: 49
+  # For deploying kubecost in a cluster that does not self-monitor
+  etlReadOnlyMode: true
```

Comment:
We don't want to default this to true, right?

Reply:
Good catch, holdover from testing...